tests/dbgp/dbgpclient.php

```diff
@@ -33,9 +34,14 @@ public function __construct()
 		$this->tmpDir = sys_get_temp_dir();
 	}
 
-	private function open()
+	private function open(&$errno, &$errstr)
```

Comment:
The Coding Standards I use have a space after ``(`` and before ``)``, as well as the ``{`` on a new line (see line 40 below).